### PR TITLE
MIS feedback

### PR DESCRIPTION
**Pull request by henryjiang-sfgov**
This PR includes all of MIS feedback from the last month.  There will be more MIS feedback in the next weeks. I'd like to establish a baseline.

Major changes:
- new module to process result file from DBI, includes a new folder ./exported_data
- update a bunch of field mappings, captured in field_maps.py, field_configs.py, export_submissions.py, and tranform.py
- updated permit_applications.py to update submission status through the building permit API

### CODE REVIEW

File: service/modules/process_result.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+ERROR_ACCESS_401 = "Unauthorized"
+
+class ProcessResultFile():
+    """ Class for processing result file """
+    data_file_path = os.path.dirname(__file__) + '/exported_data/'
+
+    def on_get(self, req, resp):
+        """ on get request """
+        try:
+            if req.params['token'] != os.environ.get('EXPORT_TOKEN') and req.params['token'] != os.environ.get('ACCESS_KEY'):
```
Comment (josh-chou) [inline service/modules/process_result.py:27]: Can't understand what is going on here.  It seems weird to check token != EXPORT_TOKEN AND token != ACESS_KEY. the only way this statement would be true is if the EXPORT_TOKEN and ACESS_KEY are the same value, and token is set to it.

File: service/modules/process_result.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+                raise ValueError(ERROR_ACCESS_401)
+
+            timezone = pytz.timezone('America/Los_Angeles')
+            current_time = datetime.datetime.now(timezone)
+            file_name = 'DBI_permits_' + str(current_time.year) + str(current_time.month) + str(current_time.day)
+            #DBI_permits_YYYYMMDDHHMI_response.csv  where HH = 24 hour clock Mi  = minutes
+            file_name = 'PTS_Export_09_26.csv'
```
Comment (josh-chou) [inline service/modules/process_result.py]: Is this value hardcoded for testing?

File: service/modules/process_result.py [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+            if fields[0] == 'FORMIO':
+                continue
+            formio_id = fields[0]
+            status = fields[1]
+            if formio_id in exported_submissions:
+                tracker['formio id'].append(formio_id)
+                tracker['status'].append(status)
+                tracker['error'].append(fields[2])
+                #file2.write(str(exported_submissions[formio_id]))
+                street_num = exported_submissions[formio_id]['projectAddressNumber'] if 'projectAddressNumber' in exported_submissions[formio_id] else ''
```
Comment (josh-chou) [inline service/modules/process_result.py:95]: can be simplified to street_num = exported_submission[formio_id].get('projectAddressNumber', '')

File: service/modules/process_result.py (hunk 2 above)
Comment (henryjiang-sfgov) [reply service/modules/process_result.py]: yes, we're waiting for final confirmation from MIS

File: service/modules/process_result.py (hunk 3 above)
Comment (henryjiang-sfgov) [reply service/modules/process_result.py:95]: Good idea.  These field gets will be moved to a helper function in the next sprint(epr-246 and epr-253).

**Comment by henryjiang-sfgov** [comment]
@josh-chou please review, I've completed the tests, at least the baseline for them.

**Comment by josh-chou** [comment]
> @josh-chou please review, I've completed the tests, at least the baseline for them.

latest changes look fine.  don't forget to remove the hardcoded file name in process_result.py before merging to main.

**Pull request merged**